core/modules/user/store/actions.ts

```diff
@@ -271,10 +271,10 @@ const actions: ActionTree<UserState, RootState> = {
       }
     }
   },
-  sessionAfterAuthorized ({ dispatch }, { refresh = onlineHelper.isOnline, useCache = true }) {
+  async sessionAfterAuthorized ({ dispatch }, { refresh = onlineHelper.isOnline, useCache = true }) {
     Logger.info('User session authorised ', 'user')()
-    dispatch('me', { refresh, useCache })
-    dispatch('getOrdersHistory', { refresh, useCache })
+    await dispatch('me', { refresh, useCache })
```

Comment:
shouldn't it be executed in parallel without await?

Reply:
not really because `me` triggers other actions, such us `cart/authorize` I remember I had problem with that, because that `cart/authorise` must be triggered before making the order... but let me check again, maybe it's really unnecessary here.